src/h5web/providers/jupyter/models.ts

```diff
@@ -41,3 +41,9 @@ export type JupyterContentResponse =
 export type JupyterDataResponse = HDF5Value;
 
 export type JupyterAttrsResponse = Record<string, string>;
+
+export type JupyterComplex = JupyterComplexArray | JupyterComplexValue;
+export type JupyterComplexValue =
+  | `(${'' | '-'}${number}${'+' | '-'}${number}j)`
+  | `${'' | '-'}${number}j`;
+export type JupyterComplexArray = JupyterComplex[];
```

Comment:
Same, I don't think having `JupyterComplexArray` is required. `JupyterComplex[]` works just as well.